backend/FwHeadless/Controllers/MediaFileMetadataController.cs

```diff
@@ -0,0 +1,23 @@
+using LexCore.Entities;
+using LexData;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace FwHeadless.Controllers;
+
+public static class MediaFileMetadataController
+{
+    [HttpGet("/{fileId}")]
```

Comment:
this attribute isn't doing anything anymore

Reply:
Done.